spring-cloud-gcp-pubsub/src/test/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapterTests.java

```diff
@@ -126,59 +119,60 @@ public void testAckModeAutoAck() throws Exception {
 				this.pubSubSubscriberOperations, "testSubscription");
 
 		adapter.setAckMode(AckMode.AUTO_ACK);
-
-		when(this.pubSubSubscriberOperations.subscribe(anyString(), any(MessageReceiver.class))).then(this.testAnswer);
-
 		adapter.setOutputChannel(this.messageChannel);
 
+		doAnswer(new CallbackAnswer(this, NACK)).when(
+				this.ackReplyConsumer).nack();
```

Comment:
You're stubbing the same method with the same answer.
Actually with Mockito you can't re-stub at all. That's not how things work there, although it may pass for you, but may fail for me and so on. Right, it is a limitation, but still Mockito is very powerful tool.
You have to move this stubbing into the `setUp()`.

Reply:
Fixed.